From: Alec Leamas <leamas.alec@gmail.com>
Date: Sun, 19 Jul 2020 19:54:28 +0200
Subject: CMakeLists: Adjust udev rule install path (#1988).

Move udev rule from /etc/udev to /lib/udev.

Bug: https://github.com/OpenCPN/OpenCPN/issues/1988
Applied-Upstream: https://github.com/OpenCPN/OpenCPN/commit/0e574866f63c
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index e4b73bc..e145100 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -2228,7 +2228,7 @@ if (NOT APPLE)
   IF(PACKAGE_FORMAT MATCHES "DEB")
         INSTALL(FILES "buildlinux/98-sglock-ocpn.rules"
           PERMISSIONS OWNER_READ OWNER_WRITE GROUP_READ WORLD_READ 
-          DESTINATION "/etc/udev/rules.d")
+          DESTINATION "/lib/udev/rules.d")
   ENDIF(PACKAGE_FORMAT MATCHES "DEB")
 
 endif (NOT APPLE)
